src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -55,7 +55,7 @@ type ChildrenArgs = {
   isOpen: boolean;
   selectedItem?: Item;
   selectedItemIndex?: number;
-};
+} & Record<string, any>; //Record is needed here because of the projectSelector component as it passes the selectedProjects prop
```

Comment:
Does this generally just break type checking on the ChildrenArgs?

Reply:
yes :)

Reply:
yes, I've added this only because the `projectSelector` component is passing a prop called `selectedProjects` down to the children, and without it, the UI breaks. We should definitely review the `multipleProjectSelector` and `projectSelector` in the future.